clarifai_model_utils/llm_eval/evaluator/harness_eval/Readme.md

```diff
@@ -11,6 +11,24 @@ Other features include:
 - Fetching documentation per template
 - Accessing configuration per template
 
+#### 1.1. Wrapper
+The wrapper is located in `ClarifaiModelHarnessEval.call_harness_eval` method.
+
+```python
+results = evaluator.evaluate(
+        lm=lm,
+        task_dict=task_dict,
+        ...
+    )
+```
+where:
+* lm: llm model derived from [lm_eval.api.model.LM](https://github.com/EleutherAI/lm-evaluation-harness/blob/main/lm_eval/api/model.py)
+* task_dict: Dict of task name and [lm_eval.api.task.Task](https://github.com/EleutherAI/lm-evaluation-harness/blob/main/lm_eval/api/task.py)
+
+#### 1.2. Dataset
+
+Since the initial goal is to evaluate model and dataset uploaded in the platform. The harness-eval consumes hf dataset to run, so we first download Clarifai dataset to temporary file and assign it to harness-eval the config (Task) see [here](https://github.com/Clarifai/clarifai-model-utils/blob/main/clarifai_model_utils/llm_eval/evaluator/harness_eval/evaluate.py)
```

Comment:
> harness-eval consumes hf dataset to run

it's good that we now support running with HF datasets as well as we are not a dedicated host for most datasets.